### DISCUSSION TITLE
--config-file in nuxt3

### DISCUSSION THREAD

**Discussion by nepster-web** [nuxt/nuxt#26455] [Questions]
in the previous version I could specify a configuration file:

```
...
"client-dev": "nuxt --config-file /app/client.nuxt.config.js",
"client-build": "nuxt build --config-file /app/client.nuxt.config.js",
"client-start": "nuxt start --config-file /app/client.nuxt.config.js",
"control-dev": "nuxt --config-file /app/control.nuxt.config.js",
"control-build": "nuxt build --config-file /app/control.nuxt.config.js",
"control-start": "nuxt start --config-file /app/control.nuxt.config.js",
...
```

but it doesn't work for nuxt3. why did it happen ?

**Comment by jwhazel** [comment]
I think it was done to allow more control while still maintaining cohesion. 
* If you just need to override some settings per environment, you can do that in a single config: https://nuxt.com/docs/getting-started/configuration#environment-overrides
* If you need a whole new config for a sibling project, like in a monorepo, use layers: https://nuxt.com/docs/getting-started/layers
